README.rst

```diff
@@ -67,44 +66,77 @@ information::
 
 
-Using autofixtures as tool for unittests
+Using autofixtures as a tool for unittests
 ========================================
 
-It has proofed that autofixtures have a great use for unittests. It has always
-bugged me that creating complex models for testing their behaviour was
-complicated. Sometimes models have strict restrictions or many related objects
-which they depend on. One solution would be to use traditional fixtures
-dumped from your production database. But while in development when database
-schemes are changing frequently, its hard to maintain all fixtures and to know
-exactly which objects are contained in the dumps etc...
+Testing the behavior of complex models has always bugged me. Sometimes models 
+have many restrictions or many related objects which they depend on. One solution 
+would be to use traditional fixtures dumped from your production database. But 
+while in development when database schemes are changing frequently, it can be time
+consuming and sometimes difficult to deep track of changes and what each dump contains.
 
-Autofixtures to the rescue! It lets you automatically generate models and all
-of their dependencies on the fly. Have a look at the following examples.
+Autofixtures to the rescue! 
 
-Lets start with the very basics. We create an ``AutoFixture`` instance for the
+Let's start with the basics. We create an ``AutoFixture`` instance for the
 ``Entry`` model and tell it to create ten model instances::
 
+(in Python shell)
+
     from autofixture import AutoFixture
     fixture = AutoFixture(Entry)
     entries = fixture.create(10)
 
-Now you can play around and test your blog entries. By default dependencies of
-foreignkeys and many to many relations are solved by randomly selecting an
-already existing object of the related model. What if you don't have one yet?
+Generic Example:
+
+    from autofixture import AutoFixture
+    fixture = AutoFixture(<replace with your model name>)
+    entries = fixture.create(10) #10 or the number of entries you would like
+
+Here are further examples for newer developers.  
+
+I have a Listing model and I want it populated with 10 objects.
+
+    from autofixture import AutoFixture
+    fixture = AutoFixture(Listing)
+    entries = fixture.create(10) 
+
+Here I've added field valules which allow you to default a field to 
+a certain value rather than the random entries supplied by *autofixture*.
+
+Generic Example including field_values:
+    
+    from <yourapp>.models import <your model>
+    fixture = AutoFixture(<your model>, field_values={‘<your field name>’:<value>})
+
+Specific example:
+
+    from main.models import Listing
+    fixture = AutoFixture(Listing,field_values={'needed_players':(randint(2,10))})
+    entries=fixture.create(30) 
+
+In the above, I wanted the 'needed_players' (in the Session model) to have only 
+numbers between 2 and 10, but I could have put {'needed_players':5} if I had wanted 
+all 'needed_players' instances to be 5.  
```

Comment:
The datastructures like `'needed_players'` and the dictionary should be marked up like code:

```
... have put ``{'needed_players': 5}`` if I had ...
```